docs/developer-docs/latest/plugins/graphql.md

```diff
@@ -891,3 +891,19 @@ mutation {
 :::
 
 Then on each request, send along an `Authorization` header in the form of `{ "Authorization": "Bearer YOUR_JWT_GOES_HERE" }`. This can be set in the HTTP Headers section of your GraphQL Playground.
+
+### API tokens
+
+To use API tokens for authentication pass the token in the `Authorization` header using the format `bearer your-api-token`.
+
+:::note
+Using API tokens in the the GraphQL playground requires a slightly different syntax. In the `HTTP HEADERS` tab add the following code:
```

Comment:
```suggestion
Using API tokens in the the GraphQL playground requires adding the authorization header with your token in the `HTTP HEADERS` tab:
```

The syntax should be the same as if the graphql queries are being made programmatically, so maybe something like this?